src/Controller/PlanningController.php

```diff
@@ -69,6 +71,34 @@ public function holidayPlanning(Request $request): Response
         return $this->createResponse('holiday', $data);
     }
 
+    #[Route('/sync-issues', name: 'app_planning_issues_sync', methods: ['POST'])]
+    public function syncAllIssues(Request $request, DataSynchronizationService $dataSynchronizationService, ProjectRepository $projectRepository): Response
+    {
+        $projectId = $request->query->get('id');
+
+        if (null === $projectId) {
+            throw new BadRequestHttpException('Project query parameter "id" not set');
+        }
+
+        $project = $projectRepository->find($projectId);
+
+        if (null === $project) {
+            throw new BadRequestHttpException('Project not found');
+        }
+
+        $dataProvider = $project->getDataProvider();
+        if (null === $dataProvider) {
+            throw new BadRequestHttpException('Project data provider not set');
+        }
+
+        $issuesSynced = 0;
+        $dataSynchronizationService->syncIssuesForProject((int) $projectId, $dataProvider, function () use (&$issuesSynced) {
+            ++$issuesSynced;
+        });
```

Comment:
What is the potential max execution time of the `syncIssuesForProject()` call relative to our standard `PHP_MAX_EXECUTION_TIME=30`?